redis.conf

```diff
@@ -1330,10 +1330,33 @@ disable-thp yes
 
 appendonly no
 
-# The name of the append only file (default: "appendonly.aof")
+# The base name of the append only file.
+# 
+# Redis can persist the data to a set of append-only files, they are divided
+# into two types, one is the the base type, it represents an initial (RDB
+# serialized) snapshot of the data present when the AOF is rewritten. The others
+# are incremental type, they contain the incremental commands since the last AOF
+# rewrite.
+# 
+# In order to manage these AOF files, Redis uses a manifest file to track them.
+# 
+# These files mentioned above have a certain naming rule. The appendfilename
+# config will be used as the base part of the file name, which will be added
+# suffixes to denote role and sequence number.
+# 
+# For example: 
+# The base could be: appendonly.aof_1.base
+# The incremental ones: appendonly.aof_1.incr, appendonly.aof_2.incr
+# manifest file: appendonly.aof_manifest
 
 appendfilename "appendonly.aof"
 
+# The directory name of the append only files.
+#
+# All AOF files and manifest file will be placed in this directory.
```

Comment:
commenting here, for a discussion about another block in the config that i can't comment on (no changes).

```
# When rewriting the AOF file, Redis is able to use an RDB preamble in the
# AOF file for faster rewrites and recoveries. When this option is turned
# on the rewritten AOF file is composed of two different stanzas:
#
#   [RDB file][AOF tail]
#
# When loading, Redis recognizes that the AOF file starts with the "REDIS"
# string and loads the prefixed RDB file, then continues loading the AOF
# tail.
aof-use-rdb-preamble yes
```

first, the term "preamble" is no longer applicable, since they're now in different files.
but obviously we can't rename the config.
what we can do, is update the documentation, which should state that this config determines if the REWRITE will generate an RDB format, or an AOF (RESP) format.

Reply:
# The BASE AOF encode type.
# 
# When rewriting the AOF file, Redis will generate a new INCR file and a 
# BASE file. INCR file is always AOF (RESP) encoded , but BASE file can be 
# in AOF format and RDB format. When this option is turned on, the BASE file
# use RDB format (the file name uses .rdb as the suffix), otherwise use AOF 
# format (the file name uses .aof as the suffix).
# 
# When loading, Redis recognizes that the BASE AOF file starts with the "REDIS"
# string and loads the BASE RDB file, then continues loading the INCR AOF
# file.
aof-use-rdb-preamble yes